examples/applications/ccm/aws-helm/helm-chart.yaml

```
@@ -15,7 +15,7 @@ spec:
         - --use-service-account-credentials=true
         - --configure-cloud-routes=false
       image: |-
-        tag: ${ trimSuffix "+rke2r1" .ClusterValues.Cluster.spec.topology.version }
+        tag: v1.32.1
```

```suggestion
        tag: ${ trimSuffix "+rke2r1" .ClusterValues.Cluster.spec.topology.version }
```
We currently use v1.31.4 in CI, this version should always be consistent with the cluster version. It is up to user to select version which works for them.

---

Ah, I think the bump was merged by mistake. It didn't have nightly run attached to it. I opened a fix: https://github.com/rancher/turtles/pull/1338#issue-3031611102, thanks for noticing it.

---

Yes it was, there was no run for it